libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -256,6 +256,7 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     }
 
     float current, consumed_mah, consumed_wh;
+    uint32_t time_remaining;
```

Comment:
We usually try to minimise variable scope; this should be declared further down.

Reply:
How should I do it?